build.gradle.kts

```diff
@@ -217,7 +217,7 @@ tasks.withType<Checkstyle>().configureEach {
 
 dependencies {
     testImplementation("org.apache.commons:commons-dbcp2:2.8.0")
-    testImplementation("com.amazonaws:aws-java-sdk-rds:1.11.875")
+    testImplementation("com.amazonaws:aws-java-sdk-rds:1.12.70")
```

Comment:
@hsuamz do we need to update our SDK version? If so we should probably update all of the dependencies.